packages/neutrino-preset-node/package.json

```diff
@@ -15,6 +15,7 @@
   "homepage": "https://neutrino.js.org",
   "bugs": "https://github.com/mozilla-neutrino/neutrino-dev/issues",
   "dependencies": {
+    "fast-async": "6.2.2",
```

Comment:
Not sure how this dependency was added without semver ranges. Did you add this via `yarn deps:add neutrino-preset-node fast-async`? Do you have `--exact` set somewhere?